courseaffils/columbia.py

```diff
@@ -145,7 +145,7 @@ def to_string(cdict):
     @staticmethod
     def to_dict(wind_string):
         wind_match = re.match(
-            ('t(?P<term>\d).y(?P<year>\d{4}).s(?P<section>\d{3})'
+            ('t(?P<term>\d).y(?P<year>\d{4}).s(?P<section>\w{3})'
```

Comment:
* WindTemplate test added.
* SectionKeyTemplate updated to accept alphanumeric section keys.